apps/mobile/src/components/GalleryTabs/GalleryTabsContainer.tsx

```diff
@@ -0,0 +1,72 @@
+import { useColorScheme } from 'nativewind';
+import { ForwardedRef, forwardRef, Suspense, useRef } from 'react';
+import { Tabs } from 'react-native-collapsible-tab-view';
+import {
+  CollapsibleRef,
+  TabReactElement,
+} from 'react-native-collapsible-tab-view/lib/typescript/src/types';
+
+import colors from '~/shared/theme/colors';
+
+type GalleryTabsContainerProps = {
+  children: TabReactElement<string> | TabReactElement<string>[];
+  Header: () => JSX.Element;
+};
+
+type GalleryTabsContainerType = CollapsibleRef<string>;
+
+const GalleryTabsContainer: React.FC<GalleryTabsContainerProps> = (
```

Comment:
thinking about it more, want to make sure I understand its purpose based on usage:

what's the relationship between `GalleryTabsContainer` and `GalleryTabBar`? seems related and I see we can pass in `GalleryTabBar` into `GalleryTabsContainer` as a prop, but a bit confused since the usage between the `CommunityView` and `ProfileView` looks a bit different.

How come in `ProfileView`, we pass in the `ProfileViewHeader` as the `Header` prop to `GalleryTabsContainer`, but in `CommunityView`, we pass in `GalleryTabBar` instead of `CommunityHeader` as the `Header` prop to `GalleryTabsContainer`? maybe its a naming thing and im misunderstanding some relationships but it might be easier to reason about if we can consistently pass in things like `ProfileViewHeader`, or `CommunityHeader`, as the Header prop 🤔

attaching screenshots of relevant code

![CleanShot 2023-08-02 at 10 07 52](https://github.com/gallery-so/gallery/assets/80802871/002d38ea-bf45-42c4-be61-ffc2dd1fdd56)


![Uploading CleanShot 2023-08-02 at 10.06.27.png…]()

Reply:
Basically, this is what `GalleryTabBar` and `GalleryTabsContainer` looks alike

![CleanShot 2023-08-03 at 00 05 36@2x](https://github.com/gallery-so/gallery/assets/4480258/6382318e-a0fb-4196-8869-42bcb9f36b2d)

Since, in the profile we have this component, I moved it to our own component to make it consistent. 

Agree with your insight! I've updated how we passed/used the `GalleryTabBar` in the `CommunityView` to act just like in `ProfileView`. Its much easier to understand